jolie/src/main/java/jolie/Interpreter.java

```diff
@@ -268,13 +269,18 @@ public CorrelationSet correlationInitializer() {
 
 	private final String logPrefix;
 	private final Tracer tracer;
+	public final static int bitsForServiceIdentifier = 31;
```

Comment:
Remove this and use Integer.MAX_VALUE below instead.

Reply:
But I am using this number for shifting bits